code/datums/components/slippery.dm

```diff
@@ -0,0 +1,27 @@
+/datum/component/slippery
+	var/intensity
+	var/lube_flags
+	var/mob/slip_victim
+
+/datum/component/slippery/New(datum/P, _intensity, _lube_flags = NONE)

```

Comment:
I know this is probably obvious, but why are you using New() instead of Initialize()